File: backup_before_industrialisation_20250626/backend/components/metiers/banque_finance/utils/helpers/core/utils_helper.js
// utils_helper.js - Fonctions utilitaires avancées pour Banque_Finance (JS)

module.exports = {
  formatDate: (date) => new Date(date).toISOString(),
  isObject: (obj) => obj !== null && typeof obj === 'object' && !Array.isArray(obj),
  deepClone: (obj) => JSON.parse(JSON.stringify(obj))
};

File: backup_before_industrialisation_20250626/backend/components/metiers/arts/utils/helpers/core/utils_helper.py
"""
utils_helper.py - Fonctions utilitaires avancées pour Arts (Python)
"""

import copy
from datetime import datetime


def format_date(date):
    return datetime.fromisoformat(date).isoformat()


def is_object(obj):
    return isinstance(obj, dict)


def deep_clone(obj):
    return copy.deepcopy(obj)
